### DISCUSSION TITLE
Review PR for guard-rubocop

### DISCUSSION THREAD

**Discussion by desheikh** [rubocop/rubocop#10044] [Maintenance]
Apologies for the bump here, however we have a small PR pending for a few months that is of interest to a few people.
Would appreciate it if someone could take a look: https://github.com/rubocop/guard-rubocop/pull/48 🙏

**Comment by bbatsov** [comment]
Will do! Somehow I totally missed the messages there.

**Comment by koic** [comment]
guard-rubocop 1.5.0 has been released with rubocop/guard-rubocop#48 feature!
https://github.com/rubocop/guard-rubocop/releases/tag/v1.5.0
